publish-scripts/shared/constants.py

```diff
@@ -1,7 +1,7 @@
 #! /usr/bin/env python3
```

Comment:
I don't think this should be checked: "My changes do not require documentation changes"